hack/scripts/olm-nightly-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! ${IIB_ID} ] ;
+then
+    echo -e "\nPlease set the environment variables IIB_ID and NEW_VER\n"   
+    exit
+fi
+
+INDEX="brew.registry.redhat.io/rh-osbs/iib:$IIB_ID"
+
+#patch to disable default catalog source
+oc patch operatorhub.config.openshift.io/cluster -p='{"spec":{"disableAllDefaultSources":true}}' --type=merge
+
+# apply image content source policy
+
+cat << EOF | oc apply -f - 
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy   
+metadata:  
+    name: brew-registry  
+spec: 
+    repositoryDigestMirrors: 
+    - mirrors: 
+      - registry.redhat.io/rh-sso-7/sso75-openshift-rhel8@sha256:d5829e880db4b82a50a4962d61ea148522a93644174931b256d7ad866eadcf40 
```

Comment:
This could be removed as it did not fix the issue of missing image in brew registry and we have moved to sso v7.6